wp-seo-main.php

```diff
@@ -260,9 +260,10 @@ function wpseo_init() {
  * Used to load the required files on the plugins_loaded hook, instead of immediately.
  */
 function wpseo_frontend_init() {
-	add_action( 'init', 'initialize_wpseo_front' );
+	WPSEO_Frontend::get_instance();
```

Comment:
This was changed so that`wpseo_frontend_init` doesn't get called until `init`, so this should be valid. See https://github.com/Yoast/wordpress-seo/pull/3700/files#diff-b77e38465f872c257610ba853807a43eR341